docs/api/reorder.md

```diff
@@ -73,6 +73,29 @@ import UpdatingData from '@site/static/usage/v8/reorder/updating-data/index.md';
 
 <UpdatingData />
 
+## Event Handling
```

Comment:
I don't love these section headings, but I decided to be consistent with how [Range](https://ionicframework.com/docs/api/range#event-handling) documents its events. If we want to change this I can update both components.